src/domain/event/__tests__/EditEventPage.test.tsx

```diff
@@ -145,11 +145,14 @@ test('returns to create occurrences page when it should after saving', async ()
     name: /tapahtuman nimi \(fi\)/i,
   });
   await user.type(nameField, 'fixme'); // FIXME: https://github.com/testing-library/user-event/discussions/970
+  await actWait();
   await user.clear(nameField);
+  await actWait();
   await waitFor(() => {
     expect(nameField).toHaveValue('');
   });
   await user.type(nameField, 'Testinimi');
+  await actWait();
```

Comment:
actWait should not be needed here. It can help in end of the test. https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning

Reply:
Removed most uses of `await actWait();` in 5fe6477e98db1ed3a2d774936736a99dc66aa63f